src/store/keyboard/context.js

```diff
@@ -0,0 +1,44 @@
+import { createContext, useContext, useState } from 'react';
+
+const KeyboardContext = createContext();
+
+function isOptionKey(e) {
+  return e.key === 'Alt';
+}
+
+export const KeyboardProvider = ({ children }) => {
+  const [isOptionKeyPressed, setIsOptionKeyPressed] = useState(false);
+
+  const initialValue = {
+    isOptionKeyPressed,
+  };
+
+  return (
+    <KeyboardContext.Provider value={initialValue}>
+      <div
+        role="button"
+        tabIndex={0}
+        onKeyDown={(e) => {
```

Comment:
there is a discussion pending with neha regarding `role= button`, when it is done I will raise a PR for that if this way of doing it is wrong

Reply:
@vinayak-trivedi Could you please add the ticket that is added for this?